online-boutique-tenant/kustomize/tenant.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: capsule.clastix.io/v1beta1
+kind: Tenant
+metadata:
+  name: online-boutique
+  labels:
+    app.kubernetes.io/type: prod
+spec:
+  owners:
+  - name: oidc:guest@k8saas.io
+    kind: User
```

Comment:
group to be created for the tenant. I will work on this on ccl-consulting/k8saas#660